src/client/core/utils/key-char.js

```diff
@@ -131,11 +131,8 @@ export function getArrayByKeyCombination (keysCombination) {
     //NOTE: we should separate symbol '+' that concats other keys and key '+' to support commands like the 'ctrl++'
     var keys = keysCombination.replace(/^\+/g, 'plus').replace(/\+\+/g, '+plus').split('+');
 
-    $.map(keys, function (key, index) {
-        keys[index] = key.replace('plus', '+');
-    });
 
-    return keys;
+    return keys.map(key => key.replace('plus', '+'));
```

Comment:
Don't use `Array.prototype` methods (like `some`, `map`, etc.). They can be overriden by the client code. You can implement their standalone versions in utls.